packages/create-keystone-app/package.json

```diff
@@ -17,6 +17,7 @@
     "@samverschueren/stream-to-observable": "^0.3.0",
     "arg": "^4.1.0",
     "chalk": "^2.4.2",
+    "cross-env": "^5.2.0",
```

Comment:
```suggestion
```

This isn't needed here (it's meant to be in `packages/create-keystone-app/templates/todo/package.json.ejs`)